### PR TITLE
feat: use iotajs to generate legacy addresses from a legacy seed

### LINKED ISSUE #39: [Task]: Adapt `getMigrationData`: Use `iotajs` to generate legacy addresses from a legacy seed
> https://github.com/iotaledger/iota.js/blob/ed5c1638af5a15e4b7d5520128b318f3eeed3da0/packages/core/src/createGetNewAddress.ts#L96
So import "@iota/core": "^1.0.0-beta.30"https://www.npmjs.com/package/@iota/core in package.json and then just do
const core = require('@iota/core');
const seed = '9'.repeat(81)
//Params: seed, index, seclvl, checksum
console.log(core.generateAddress(seed, 0, 2, false))
// GPB9PBNCJTPGFZ9CCAOPCZBFMBSMMFMARZAKBMJFMTSECEBRWMGLPTYZRAFKUFOGJQVWVUPPABLTTLCIA

- [ ] Generate address using iotajs
- [ ] Convert the generated address to binary format so its like `0x....`, as explained here https://github.com/iotaledger/wasp/blob/legacy-migration/README.md#testing-instructions-localhost

### PR DESCRIPTION
**Pull request by cpl121**
Closes #39 

## Summary
Use iotajs to generate legacy addresses from a legacy seed

### Changelog
```
Use iotajs to generate legacy addresses from a legacy seed.
```

## Type of Change
Please select any type below that is applicable to your changes, and delete those that are not.
- [ ] __Breaking__ - any change that would cause existing functionality to not work as expected
- [ ] __Chore__ - refactoring, build scripts or anything else that isn't user-facing
- [ ] __Docs__ - changes to the documentation
- [ ] __Fix__ - a change which fixes an issue
- [ ] __New__ - a change which implements a new feature
- [X] __Update__ - a change which updates existing functionality

### Platforms
Please select any platforms where your changes have been tested.
- __Desktop__
	- [ ] MacOS
	- [X] Linux
	- [ ] Windows

## Checklist
Please tick all of the following boxes that are relevant to your changes, and delete those that are not.
- [X] I have followed the contribution guidelines for this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [X] I have verified that new and existing unit tests pass locally with my changes
- [X] I have verified that my latest changes pass CI workflows for testing and linting
- [X] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/shared/lib/migration.ts [hunk 1]
```diff
@@ -169,6 +170,15 @@ export const getMigrationData = (migrationSeed: string, initialAddressIndex = 0)
             reject({ snapshot: true })
             openSnapshotPopup()
         } else {
+            // Generate address using iotajs
+            const legacyAddresses: string[] = []
+            const binaryAddresses: string[] = []
+            for (let index = 0; index < 10; index++) {
+                const legacyAddress = generateAddress(migrationSeed, index, 2)
```
Comment (begonaalvarezd) [inline packages/shared/lib/migration.ts:177]:
```suggestion
                const legacyAddress = generateAddress(migrationSeed, index, ADDRESS_SECURITY_LEVEL)
```

Comment (begonaalvarezd) [reply packages/shared/lib/migration.ts:177]: Also, based on what was shared in the slask channel, we might have to remove the last 9 characters as they are the checksum of the old address format, i dont know if we have to remove them, but just to be aware of this

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]
Amazing 🥳 added a mini comment

File: packages/shared/lib/migration.ts (hunk 1 above)
Comment (cpl121) [reply packages/shared/lib/migration.ts:177]: It comes with the correct format of 81 characters in length

**Review by begonaalvarezd — APPROVED** [review]

**Pull request merged**